01week/datatypes.js

```diff
@@ -0,0 +1,66 @@
+`use strict`
+
+//returns current date and time
+//make a today var, which is assigned a new date instance
+//make new var to store the today var and toDateString method
+//use var today with toTimeString method
+
+var today = new Date(1985, 11, 1);
+console.log(today.toDateString());
+
+//create a function that will supply the resulting string
+//create a var to convert.
+//input a number into var convert
+//use convert and toString() method to make the number a string
+
+function toNumber() {
+  var num = 13
+  var n = console.log(num.toString)
+  return n
+}
+//set var to a string
+//use parseInt() method
+
+var str = `15 Years`
+var int = console.log(parseInt(str))
+
+//tells what type of datatype your input is
+//create a variable of any main datatype
+//use the typeOf method on the variable
+
+var dataType = `words words words`
+var result = console.log(typeof(dataType))
+
+//Create two var that can be a string or number, the string must contain a number though
+//Create a function with the two var
+//Inside the function, add the two var together
+//Use parseInt() to take the number out of the string
+
+function add(numOne, numTwo) {
+  var total =  (parseInt(numOne) + parseInt(numTwo));
+  return total('13', '17')
+}
+
+//write a function that returns a true if two var are truthy
+//if both var are true, show "nope"
+//if both var are false, also show "nope"
+// if one is false and one is true, show "yep!"
+
+function truest(thingOne, thingTwo) {
+  if (thingOne && thingTwo === `true`) {
+    return `nope`
+  } else if (thingOne && thingTwo === `false`) {
+    return 'nope'
+  } else {
+    return 'yep!'
+  }
+};
+
+//write a function that only runs when both var are false
+//if both var are false, show a confirmation message
+
+function womp(dumb, dumber) {
+  if (dumb && dumber === `false`) {
```

Comment:
This block of code will run if dumb evaluates to true and if dumber is equal to the string "false". That isn't what the problem asked. I need it to run if either dumb evaluates to false and if dumber evaluates to false.